src/apps/config/leader.lua

```diff
@@ -70,16 +69,31 @@ end
 
 function Leader:set_initial_configuration (configuration)
    self.current_configuration = configuration
-   self.current_app_graph = self.setup_fn(configuration)
    self.current_in_place_dependencies = {}
-   self.current_in_place_dependencies =
+
+   -- Start the followers and configure them.
+   local follower_configurations = self.setup_fn(configuration)
```

Comment:
It would be nice if the word "configuration" applied either to the user-visible configuration of the network service, as instance data of the YANG schema, or to the Snabb-internal configuration of the graph of applications and links.  WDYT?  If you don't have a preference as to which is which, let's reserve "configuration" for the user-visible YANG config, and use the term "app graph" or "graph" to describe the Snabb-internal graph of apps and links.  Then I guess we would call this variable "worker_app_graphs" or something